articles/client-platforms/react/05-linking-accounts.md

```diff
@@ -0,0 +1,372 @@
+---
```

Comment:
@flaviogranero 
The document does not load. I believe the reason is not escaping strings: https://github.com/auth0/docs#escaping-strings

For example, this:
`https://${this.domain}/api/v2/users/${userId}/${url}`

Should be:
`https://<%= "${this.domain}" %>/api/v2/users/<%= "${userId}" %>/<%= "${url}" %>`